### ISSUE TITLE
Sql AFTER statement

### ISSUE THREAD

**Issue by ncsft** [db-migrate/node-db-migrate#265]
I want to run something like that?
`alter table ** add column new_column int(10) after exists_column`

**Comment by wzrdtales** [comment]
@ncsft Please write a more thorough issue. This issue is completely missing any details.

What is your exact problem?
Why do you want to do, what you try to do?
How did you tried it yet?

Now it seems you have simple SQL Question, this not related to db-migrate, but to SQL instead. To answer how you execute a SQL Query within a migration, this can be done via runSql.
Please always write a detailed issue and not a one liner lacking of information, we all would like to help you, but we can't when we don't know what you want.

Thank you

**Comment by ncsft** [comment]
I want to add column "commission" exactly after column "login" in table "partner"
```
        db.addColumn.bind(db, 'partner', 'commission', {
            type: 'int',
            unsigned: true,
            notNull: true,
            defaultValue: '0'
        }),
```

**Comment by wzrdtales** [comment]
@ncsft No not yet, this is also related to #159 
This is considered to be a nice to have, as it doesn't influence the performance of anything, but the visual appearance. Thus this has a low priority, if you really want this feature to be implemented fast there are two ways.

1) Do it yourself and open a pull request afterwards.
2) Put a bounty on it, to make the issue attractive for anyone to solve it. This mustn't be me in this case and could be anyone.

**Comment by wzrdtales** [comment]
Closed as duplicate of #159

PRs welcome
